plugins/slick.rowmovemanager/slick.rowmovemanager.js

```diff
@@ -82,24 +86,26 @@ function RowMoveManager(options){
     e.stopImmediatePropagation();
 
     const top = e.pageY - $(_canvas).offset().top;
-    dd.selectionProxy.css('top', top - 5);
+    selectionProxy.css('top', top - 5);
+
+    const insertBeforeNew = Math
+      .max(0, Math.min(Math.round(top / _grid.getOptions().rowHeight), _grid.getDataLength()));
 
-    const insertBefore = Math.max(0, Math.min(Math.round(top / _grid.getOptions().rowHeight), _grid.getDataLength()));
-    if (insertBefore !== dd.insertBefore){
+    if (insertBeforeNew !== insertBefore){
       const eventData = {
-        'rows': dd.selectedRows,
-        'insertBefore': insertBefore
+        'rows': selectedRows,
+        'insertBefore': insertBeforeNew
       };
 
       if (_self.onBeforeMoveRows.notify(eventData) === false){
-        dd.guide.css('top', -1000);
-        dd.canMove = false;
+        guide.css('top', -1000);
+        canMove = false;
       } else {
-        dd.guide.css('top', insertBefore * _grid.getOptions().rowHeight);
-        dd.canMove = true;
+        guide.css('top', insertBeforeNew * _grid.getOptions().rowHeight);
+        canMove = true;
       }
 
-      dd.insertBefore = insertBefore;
+      insertBefore = insertBeforeNew;
```

Comment:
did this work as expected? because in this case you just create a link

Reply:
insertBeforeNew and insertBefore are integers. it's work correctly of course.